### DISCUSSION TITLE
`user` role is not able to perform mutations

### DISCUSSION THREAD

**Discussion by yogeshwar607** [hasura/graphql-engine#7301] [General] [answered]
`user` role is not able to perform mutations even after giving proper permissions.
Previously it was working fine and there were no issues but now even in console its not showing mutations option all together even with proper headers.

Everything is working fine with `admin` role though

<img width="1440" alt="Screenshot 2021-07-27 at 7 03 02 PM" src="https://user-images.githubusercontent.com/10059011/127164148-7657ab07-0a1a-4faf-b099-d44017f36850.png">
<img width="1440" alt="Screenshot 2021-07-27 at 7 06 04 PM" src="https://user-images.githubusercontent.com/10059011/127164175-3268b4e4-f2bf-477e-9886-b0a6d6d140ff.png">

**Comment by yogeshwar607** [comment]
I tried with new role `admin_user` , same issue. Explorer is not showing me `mutation` option and not able to perform mutation operations


<img width="1440" alt="Screenshot 2021-07-27 at 7 59 10 PM" src="https://user-images.githubusercontent.com/10059011/127172477-ba9feee2-12c8-47ac-8a7c-3e36e45cb94e.png">

<img width="1440" alt="Screenshot 2021-07-27 at 7 58 59 PM" src="https://user-images.githubusercontent.com/10059011/127172508-6ba88612-9cea-4ef3-a8c9-f03e6c8b9548.png">

**Reply by yogeshwar607** [reply]
Update  : if i remove HASURA_GRAPHQL_EXPERIMENTAL_FEATURES  :  inherited_roles and remove `inherited_roles` from `/console/settings/inherited-roles` then it works 

But `inherited_roles` is major feature and we have already designed our system surrounding it so need a fix for this 

Version : 2.0.3-cloud.1

**Comment by coco98** [comment, marked as answer]
@yogeshwar607 This will be fixed in our next release and we've already merged the code that removes inherited roles from being experimental and mutations, remote-schemas and actions are now supported!

We're doing an overview in the community call tomorrow as well:
https://hasura.io/community/community-call/

**Reply by yogeshwar607** [reply]
https://github.com/hasura/graphql-engine/issues/7304#issuecomment-889294436

still same issue in `v2.0.4`

**Reply by iosifnicolae2** [reply]
@coco98 I can't find where was released the fix (was it released only in the cloud version?)

**Reply by rikinsk** [reply]
@iosifnicolae2 the fix has been released in `v2.0.5`

**Reply by yogeshwar607** [reply]
got it working now

**Comment by iosifnicolae2** [comment]
Hmm, but I cannot use it.., basically I cannot execute mutations while
using an inherited_role (I have inherited_roles enabled as experimental
feature, if I disable it the process of applying the metadata will fail).
